api_app/analyzers_manager/observable_analyzers/intezer_get.py

```diff
@@ -0,0 +1,61 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.file_analyzers.intezer_scan import (
+    get_access_token,
+    get_intezer_base_url,
+)
+
+from tests.mock_utils import if_mock_connections, patch, MockResponse
+
+logger = logging.getLogger(__name__)
+
+
+class IntezerGet(classes.ObservableAnalyzer):
+    base_url: str = get_intezer_base_url()
+
+    def set_params(self, params):
+        self.__api_key = self._secrets["api_key_name"]
+        # intezer access token
+        self.intezer_token = get_access_token(self.__api_key)
+
+    def run(self):
+        return self.__intezer_analysis()
```

Comment:
I'd argue that the `_intezer_analysis` serves no actual purpose and adds an unnecessary overhead to the call stack, the function body can directly be part of `run` method.